docs/content/guide/theming.md

```diff
@@ -63,12 +63,12 @@ The available data provided to the template scope is determined by the relevant
 </DruxtEntity>
 ```
 
-By default, a component using the default template will not be wrapped by a DruxtWrapper component. It is possible to enable the DruxtWrapper system by setting the `wrapper` property to `true`:
```

Comment:
This is also wrong, the DruxtWrapper is the placeholder component, this hasn't changed.